### ISSUE TITLE
Debian Bookworm: Error: 0: Couldn't get terminal

### ISSUE THREAD

**Issue by axelsimon** [DeterminateSystems/nix-installer#915]
Trying to install (as root, or as a regular user) on Debian 12 (Bookworm) results in the following:

## Error
```
Error: 
   0: Couldn't get terminal
```

## Metadata
|key|value|
|--|--|
|**version**|0.17.1|
|**os**|linux|
|**arch**|x86_64|

**Comment by cole-h** [comment]
Sounds like https://github.com/DeterminateSystems/nix-installer/issues/868 -- could you try my steps in https://github.com/DeterminateSystems/nix-installer/issues/868#issuecomment-1967833471 and see if that works for you?

**Comment by axelsimon** [comment]
Indeed, thanks @cole-h. I've added a workaround in the other issue, so closing this one.
